README.md

```diff
@@ -13,44 +15,60 @@
 1. It will create a local configuration file
 
 ```bash
-git-message-ai initiate
+git-ai initiate
 ```
 
 2. Declare these envinroment varibles:
 
 ```.env
-GMA_JIRA_AUTHORIZATION=<jira authorization can be your user and password as a basic>
-GMA_OPENAI_TOKEN=<you can get it here: https://platform.openai.com/api-keys>
+
+# If you are using JIRA
+GA_JIRA_API_KEY=<you can get it here: https://id.atlassian.com/manage-profile/security/api-tokens>
+GA_JIRA_USER=<your user to sign in into jira>
```

Comment:
Now the prefix is `GM` instead `GMA` because the package name we are receiving `GA_JIRA_API_KEY` and `GA_JIRA_USER`!

Reply:
What about use [this one](https://docs.rs/config/latest/config/)?